integration/spark/src/test/scala/org/apache/spark/sql/TestCreateTable.scala

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
pls move the contents into incubator-carbondata\integration\spark\src\test\scala\org\apache\carbondata\spark\testsuite\createtable\TestCreateTableSyntax.scala